src/main/java/com/google/codeu/data/Message.java

```diff
@@ -42,6 +42,8 @@ public Message(UUID id, String user, String text, long timestamp) {
     this.timestamp = timestamp;
   }
 
+
```

Comment:
Remove newline only changes. In general, unless you are cleaning up formatting of the codebase, you shouldn't add a file for review that has whitespace only changes.